help_to_heat/frontdoor/routing/backwards_routing.py

```diff
@@ -0,0 +1,30 @@
+from help_to_heat.frontdoor.consts import (
+    country_page,
+    govuk_start_page,
+    unknown_page,
+)
+from help_to_heat.frontdoor.routing.forwards_routing import get_next_page
+
+start_page = country_page
+max_route_length = 100
+
+
+def get_prev_page(current_page, answers):
+    route_page = start_page
+    route = [govuk_start_page]
```

Comment:
Probably doesn't make a real difference with the quantities of data we're talking about, but I believe a deque would be more performant here.

Reply:
have changed to a deque